### PR TITLE
Add PDB format option to /debug flag

### PR DESCRIPTION
**Pull request by TyOverby**

### CODE REVIEW

**Comment by TyOverby** [comment]
CC @agocke @AlekseyTs @jaredpar @TyOverby @VSadov @tmat
Please review

File: src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs [hunk 1]
```diff
@@ -1139,6 +1136,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 OutputFileName = outputFileName,
                 PdbPath = pdbPath,
                 EmitPdb = emitPdb,
+                // TODO: pipe debugInformationFormat through this
```
Comment (jaredpar) [inline src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs]: What work is left here?

Comment (TyOverby) [inline src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs]: Oops.  I added that back before I learned that the format was already passed through on line 1116.  It should be removed; there's no more work left.

**Comment by jaredpar** [comment]
:+1:

**Comment by TyOverby** [comment]
I removed the todo comment.  Can I manually merge this right now?  The only change is that I removed the comment, and 4/5 tests passed on the first commit (the only one failing was a flakey ETA test).

**Pull request merged**